yax/core/src/main/scala/doobie/util/compat.scala

```diff
@@ -63,7 +63,14 @@ object compat {
 
 #+fs2
     object fs2 {
-      import _root_.fs2.util.{ ~> => Fs2Nat }
+      import _root_.fs2.util.{ Monad => Fs2Monad, ~> => Fs2Nat }
+
+      implicit def monadCompat[F[_]](implicit monad: Monad[F]): Fs2Monad[F] =
```

Comment:
Could this be replaced by bringing in `fs2-cats` and `fs2-scalaz` ? https://github.com/functional-streams-for-scala